supervisord.conf

```diff
@@ -2,7 +2,7 @@
 nodaemon=true
 
 [program:X11]
-command=/usr/bin/Xvfb :0 -screen 0 1024x768x24
+command=/usr/bin/Xvfb :0 -screen 0 1920x1080x24
```

Comment:
@leonowski 

Can we make this an environmental variable? Set the default in Dockerfile to 1920x1080x24 and then let users change that variable. This way we can support multiple screen sizes from the same container.

Reply:
I personally found `1440x768x24` worked great on my MacBook and then was a little small on my Windows PC, but still very viable.

I'd like to propose `1024x768x24` or `1440x768x24` as default, preferably the latter. I also agree on the environment variable to set it.

Reply:
Agree - smaller is probably better so everybody can see it at first launch and then they can expand it if needed. I'm working on the commits now and will push to this PR.

Reply:
Yes - sorry, ENV VAR for resolution is best.